packages/@okta/vuepress-site/docs/concepts/session/index.md

```diff
@@ -13,43 +13,47 @@ An Identity Provider (IdP) is a service that stores and manages digital identiti
+Local logout is the act of signing the user out of the app and terminating the application session. The user IdP session is still active, and other application sessions aren't affected by terminating a specific local application session. The app manages local logout. If the user wants to access the app again, the user is automatically authenticated through the active IdP session into the app, and the app creates a session for the user.
 
-See [Sign users out > Sign users out of your app](/docs/guides/sign-users-out/-/main/#sign-users-out-of-your-app) for a guide to implement local logout with Okta.
+See [Sign users out](/docs/guides/sign-users-out) for a guide to implement local logout with Okta.
 
-### Single logout
+### Single Logout (SLO)
 
 Single logout is the act of signing the user out of the IdP and signing out of all the apps with the same IdP credentials. This results in ending the IdP session and all the associated application sessions for the user.
 
-Currently, Okta supports single logout initiated by an app, where the user signs out of an app and the Okta IdP, ending both the app and IdP sessions. However, other active application sessions for the user can still persist without the IdP session, depending on the restrictive nature of the apps. For non-privileged access apps, the user can still access the app within the scope of the application session. For example, a user can still browse through a catalog of vacation experiences and add to their cart. When this user is ready to check out and make a payment, the app can enforce a privileged access workflow and redirect the user to re-authenticate with Okta to secure an IdP session. For privileged access apps with short-lived sessions, the user is typically redirected to re-authenticate with Okta to start a new IdP session and an application session.
+> **Note**: App-initiated single logout is also known as Service Provider-initiated (SP-initiated) single logout (SLO).
 
-See [Sign users out > Sign users out of Okta](/docs/guides/sign-users-out/-/main/#sign-users-out-of-okta) for a guide to implement Okta sign out. For Okta Admin Console app-initiated single logout configuration, see [Single Logout in applications](https://help.okta.com/okta_help.htm?id=ext_Apps_Single_Logout).
+Currently, Okta supports app-initiated single logout, where the user signs out of both an app and the Okta IdP to end both the app and IdP sessions. However, other app sessions for the user might persist without the IdP session, depending on the restrictive nature of the apps.
```

Comment:
I see this term used often: `currently` - does it make sense to omit this word as it sounds like it'll get dated pretty quick once idp-init SLO is not "new" anymore? Generally I feel like we should not say currently since it should be implied when we just say "okta supports"